Index: Irrlicht/CFileSystem.cpp
===================================================================
--- Irrlicht/CFileSystem.cpp	(revision 592)
+++ Irrlicht/CFileSystem.cpp	(working copy)
@@ -838,7 +838,7 @@
 		#ifdef _IRR_WINDOWS_API_
 		#if !defined ( _WIN32_WCE )
 
-		r = new CFileList(Path, true, false);
+		r = new CFileList(Path, false, false);
 
 		// TODO: Should be unified once mingw adapts the proper types
 #if defined(__GNUC__)
